### PR TITLE
WebSDK Linting & Nits

### PR DESCRIPTION
**Pull request by rgomezp**
# Description
## 1 Line Summary
Includes a couple nits and linting cleanup

## 
This change is [Reviewable](https://reviewable.io/reviews/onesignal/onesignal-website-sdk/788)

### CODE REVIEW

**Review by jkasten2 — CHANGES_REQUESTED** [review]
Let's hold off on merging this PR or basing future PRs off of this one until we have the player update refactor merged into master. The ts config fix operation is very easier to rerun again after this.

> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/onesignal/onesignal-website-sdk/788#-:-MXkAJ5X-tc2XB_JsBbB:byk233o)* status: 1 of 141 files reviewed, 1 unresolved discussion (waiting on @rgomezp)

___
*[src/helpers/ConfigHelper.ts, line 269 at r2](https://reviewable.io/reviews/onesignal/onesignal-website-sdk/788#-MXk9u_ZBrdSnm5evnXQ:-MXk9u_ZBrdSnm5evnXR:be8bezy) ([raw file](https://github.com/onesignal/onesignal-website-sdk/blob/2783cb0700096dce5d695ef555d6fcacf46aa319/src/helpers/ConfigHelper.ts#L269)):*
> ```XML
>         };
> 
>         // default autoPrompt to true iff slidedown config exists but omitted the autoPrompt setting
> ```

This commit seems specific to prompts but the rest of the PR is a more generic limiting change. Thinking this commit should be in another PR or it's own.

**Comment by rgomezp** [comment]
Fair enough. Opened a new PR for the comment in https://github.com/OneSignal/OneSignal-Website-SDK/pull/790

For future reference: 
`tslint --config tslint.json --fix --project .` is the command to run the linting process

**Pull request closed without merging**
